### PR TITLE
Paul/dark UI update 1

### PR DESCRIPTION
**Pull request by paul4702**
image

image

### CODE REVIEW

File: src/lib/atoms/icons/Icon.svelte [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <script lang="ts">
-	import Icon from 'svelte-awesome';
+	import Icon from 'svelte-awesome/components/Icon.svelte';
```
Comment (vishwajeetraj11) [inline src/lib/atoms/icons/Icon.svelte:2]: does this fix anything?
or just enhancement/refactor/reduce bundle size thing?

File: src/lib/page-components/oyster/sub-components/PaymentHistoryTable.svelte [hunk 2]
```diff
@@ -55,7 +55,7 @@
 						</td>
 						<td class={tableClasses.cell}
 							><div
-								class={cn('w-fit rounded-full px-4 py-4', {
+								class={cn('w-fit rounded-full px-4 py-4 text-[#030115]', {
```
Comment (vishwajeetraj11) [inline src/lib/page-components/oyster/sub-components/PaymentHistoryTable.svelte:58]: does hardcoding color here works well for both dark and light mode?

File: src/lib/page-components/oyster/operator/OysterOperatorJobsTableRow.svelte [hunk 3]
```diff
@@ -90,7 +90,7 @@
 </td>
 <td class={cn(tableClasses.cell, 'text-center')}>
 	<div
-		class="mx-auto rounded-full px-[31.5px] py-[10.5px] text-center text-sm capitalize text-grey-800"
+		class="mx-auto rounded-full px-[31.5px] py-[10.5px] text-center text-sm capitalize text-[#030115]"
```
Comment (vishwajeetraj11) [inline src/lib/page-components/oyster/operator/OysterOperatorJobsTableRow.svelte:93]: does hardcoding color works well in both modes?

File: src/lib/page-components/oyster/inventory/OysterInventoryTableRow.svelte [hunk 4]
```diff
@@ -150,7 +150,7 @@
 				<span slot="tooltipContent">{epochToDurationStringLong(durationLeft)}</span>
 				<span slot="tooltipIcon">
 					<div
-						class="mx-auto rounded-full px-[31.5px] py-[10.5px] text-center text-sm text-grey-800"
+						class="mx-auto rounded-full px-[31.5px] py-[10.5px] text-center text-sm text-[#030115]"
```
Comment (vishwajeetraj11) [inline src/lib/page-components/oyster/inventory/OysterInventoryTableRow.svelte:153]: does hardcoding color works well in both modes?

File: src/lib/atoms/icons/Icon.svelte (hunk 1 above)
Comment (paul4702) [reply src/lib/atoms/icons/Icon.svelte:2]: refactor.

File: src/lib/page-components/oyster/sub-components/PaymentHistoryTable.svelte (hunk 2 above)
Comment (paul4702) [reply src/lib/page-components/oyster/sub-components/PaymentHistoryTable.svelte:58]: yes, Its for the labels. Please run it on your local machine to see the UI.

File: src/lib/components/prompts/NetworkPrompt.svelte [hunk 5]
```diff
@@ -32,18 +32,16 @@
...
+				>{title}</span
 			>
 		{/if}
 
 		{#if description}
 			<p
 				class={cn(
 					'text-md mx-auto mt-2 max-w-[400px] text-center font-light leading-6',
-					isVariantWhite ? 'text-[#A8A8A8]' : 'text-[#D9DADE]'
+					isVariantWhite ? 'text-[#A8A8A8]' : 'text-[#FCFCFC]'
```
Comment (vishwajeetraj11) [inline src/lib/components/prompts/NetworkPrompt.svelte:44]: does hardcoding color works well in both modes?

File: src/lib/page-components/oyster/operator/OysterOperatorJobsTableRow.svelte (hunk 3 above)
Comment (paul4702) [reply src/lib/page-components/oyster/operator/OysterOperatorJobsTableRow.svelte:93]: yes, Its for the labels. Please run it on your local machine to see the UI.

File: src/lib/page-components/oyster/inventory/OysterInventoryTableRow.svelte (hunk 4 above)
Comment (paul4702) [reply src/lib/page-components/oyster/inventory/OysterInventoryTableRow.svelte:153]: yes, Its for the labels. Please run it on your local machine to see the UI.

File: src/lib/components/inputs/TextInputWithEndButton.svelte [hunk 6]
```diff
@@ -45,16 +22,9 @@
...
 >
-	{#if !!title.length}
-		<button class={cn(titleStyle, titleStyleInit)}>
-			{title}
-		</button>
-	{/if}
 	<input
 		id={'address-display' + id}
 		bind:value={input}
-		on:focus={onFocus}
```
Comment (vishwajeetraj11) [inline src/lib/components/inputs/TextInputWithEndButton.svelte:56]: do we not need on focus and on blur anymore?

File: src/lib/components/header/sub-components/DisconnectWalletButton.svelte [hunk 7]
```diff
@@ -26,7 +26,7 @@
 	<ChainSwitcher />
 	<ModalButton
 		{modalFor}
-		variant="whiteFilled"
+		variant="outlined"
```
Comment (vishwajeetraj11) [inline src/lib/components/header/sub-components/DisconnectWalletButton.svelte:29]: is this intentional?
or design request?

File: src/lib/components/header/sub-components/ConnectWalletButton.svelte [hunk 8]
```diff
@@ -32,7 +32,7 @@
 		<Button
 			onclick={connect}
 			size="small"
-			variant="filled"
+			variant="outlined"
```
Comment (vishwajeetraj11) [inline src/lib/components/header/sub-components/ConnectWalletButton.svelte:35]: is this intentional or design request?

**Review by vishwajeetraj11 — DISMISSED** [review]

File: src/lib/components/inputs/TextInputWithEndButton.svelte (hunk 6 above)
Comment (paul4702) [reply src/lib/components/inputs/TextInputWithEndButton.svelte:56]: yes we don't. It was code from when we wanted to impliment the animated inputs.

File: src/lib/components/header/sub-components/DisconnectWalletButton.svelte (hunk 7 above)
Comment (paul4702) [reply src/lib/components/header/sub-components/DisconnectWalletButton.svelte:29]: It is intentional and matches up the design.

File: src/lib/components/prompts/NetworkPrompt.svelte (hunk 5 above)
Comment (paul4702) [reply src/lib/components/prompts/NetworkPrompt.svelte:44]: Yes it does

File: src/lib/components/header/sub-components/ConnectWalletButton.svelte (hunk 8 above)
Comment (paul4702) [reply src/lib/components/header/sub-components/ConnectWalletButton.svelte:35]: Both apprently.

**Review by vishwajeetraj11 — APPROVED** [review]

**Review by souvikmishra — APPROVED** [review]

**Pull request merged**